eraser-map.yaml

```diff
@@ -1466,7 +1462,7 @@ layers:
                         width: [[9, 0px], [10, 0.5px], [14, 2px], [16, 4m], [17, 5m], [18, 6m]]
                         outline:
                             color: [[9, [0.827,0.827,0.827]], [13, [0.827,0.827,0.827]], [15, [1.000,0.739,0.710]]]
-                            width: [[9, 0px], [10, 1px], [14, 1px], [16, 1.5px], [17, 2px]]
+                            width: [[13, 1px], [14, 1px], [16, 1.5px], [17, 2px]]
```

Comment:
But at z9 it would be 0.7px. So this is changing the look (was no width). Suggest keeping as is unless there is a strong perf reason.